doc/ref_model/chapters/chapter05.md

```diff
@@ -60,7 +60,7 @@ The following sections detail the Cloud Infrastructure Software Profile capabili
 | infra.com.cfg.001 | CPU allocation ratio | Value  | Number of virtual cores per physical core. | `i.cap.016` |
 | infra.com.cfg.002 | NUMA alignment | Yes/No | Support of NUMA at the Host OS and virtualisation layers, in addition to hardware. | `e.cap.007` |
 | infra.com.cfg.003 | CPU pinning | Yes/No | Binds a vCPU to a physical core or SMT thread. Configured in OS and virtualisation layers.| `e.cap.006` |
-| infra.com.cfg.004 | Huge Pages | Yes/No | Ability to manage huge pages of memory. Configured in OS and virtualisation layers. | `i.cap.018` |
+| infra.com.cfg.004 | Huge pages | Yes/No | Ability to manage huge pages of memory. Configured in OS and virtualisation layers. | `i.cap.018` |
```

Comment:
```suggestion
| infra.com.cfg.004 | Huge pages | Yes/No | Ability to manage Huge pages of memory. Configured in OS and virtualisation layers. | `i.cap.018` |
```

Reply:
Hi @pgoyal01, I've changed the capitalization to huge pages (with lowercase h and p). There are a couple cases where I used different capitalization:
- in the beginning of sentences (Huge page)
- in tables where the other entries started with uppercase (Huge page)
- in titles where title case was used (Huge Page)